test/time_stack_item_test.rb

```diff
@@ -222,7 +222,7 @@ def test_set_scaling_factor_for_scale
 
   def test_parse_only_string_with_active_support
     Time.expects(:parse).never
-    Timecop.freeze(2011, 01, 02, hour=0, minute=0, second=0)
+    Timecop.freeze(2011, 01, 02, 0, 0, 0)
```

Comment:
As part of this cleanup, note that `01`, `02`, etc are actually **octal** numbers!! I.e. `09` is invalid syntax, and `010 == 8`! It would be good to clean these up too.

Reply:
Good call.